core/message-tags-3.3.md

```diff
@@ -80,6 +89,24 @@ Individual tag keys MUST only be used a maximum of once per message. Clients
 receiving messages with more than one occurrence of a tag key SHOULD discard all
 but the final occurrence.
 
+### Empty tagged messages
+
+Servers MUST accept `PRIVMSG` and `NOTICE` events sent with client-only tags even if
```

Comment:
Right, but scope of this change is smaller.
I mean, all implementations which use some analog of `std::string`, don't need to change the whole codebase to `std::optional<std::string>` anymore. The same for `""` vs `None` in Python